examples/plot_callosal_tract_profile.py

```diff
@@ -0,0 +1,558 @@
+                    mapping.transform_inverse(
+                        roi.get_fdata().astype(np.float32),
+                        interpolation='linear'),
+                    bundle_name=bundle)
+
+                nib.save(nib.Nifti1Image(warped_roi.astype(float), img.affine),
+                         op.join(working_dir, f"{bundle}_{idx+1}.nii.gz"))
+
+                warped_roi_img = nib.load(op.join(working_dir,
+                                                  f"{bundle}_{idx+1}.nii.gz"))
+                show_anatomical_slices(warped_roi_img.get_fdata(),
+                                       f'warped {bundle}_{idx+1} ROI')
+
+                # Add voxels that aren't there yet:
+                seed_roi = np.logical_or(seed_roi, warped_roi)
+
+    seed_roi_img = nib.Nifti1Image(seed_roi.astype(float), img.affine)
+    nib.save(seed_roi_img, op.join(working_dir, 'seed_roi.nii.gz'))
+
+    show_anatomical_slices(seed_roi_img.get_fdata(), 'Seed ROI')
+
+    tractogram = aft.track(dti_params['params'], seed_mask=seed_roi,
+                           stop_mask=FA_data, stop_threshold=0.1)
+    save_tractogram(tractogram, op.join(working_dir, 'dti_streamlines.trk'),
+                    bbox_valid_check=False)
+
+    create_density_map(tractogram, 'afq_dti_density_map.nii.gz')
+else:
+    tractogram = load_tractogram(op.join(working_dir, 'dti_streamlines.trk'),
+                                 img)
+
+tractogram.to_vox()
+
+##########################################################################
+# Segmentation:
+# -------------
+# In this stage, streamlines are tested for several criteria: whether the
+# probability that they belong to a bundle is larger than a threshold (set to
+# 0, per default), whether they pass through inclusion ROIs and whether they do
+# not pass through exclusion ROIs.
+#
+# .. note::
+#   By default segmentation:
+#
+#   - uses Streamlinear Registration algorithm
+#
+#   - does not clip streamlines to be between ROIs
+#
+#   - All b-values less than or equal to 50 are considered to be
+#     without diffusion weighting.
+#
+# 
+# Segmentation will result in a `fiber_group` for each bundle, which
+# contains the following keys:
+#
+#    - `sl`
+#
+#       StatefulTractogram encompassing the selected streamlines
+#
+#    - `idx`
+#
+#       indexes to selected streamlines
+#
+# .. note::
+#    Currently it is not possible to define endpoint filters for novel bundles,
+#    but this is something we expect to address. However we can run
+#    segmentation by ignoring endpoint filters. This means that additional
+#    streamlines may be included that would otherwise be excluded.
+
+
+tractogram_img = nib.load(op.join(working_dir, 'afq_dti_density_map.nii.gz'))
+show_anatomical_slices(tractogram_img.get_fdata(), 'DTI Density Map')
+
+print("Segmenting fiber groups...")
+
+segmentation = seg.Segmentation(return_idx=True,
+                                filter_by_endpoints=False)
```

Comment:
Currently bundles_to_all hard codes the endpoints. Should be able to have users define as necessary for custom ROIs.

Reply:
Addressed in #497